admin-dev/themes/new-theme/js/components/form/change-password-control.ts

```diff
@@ -249,6 +249,7 @@ export default class ChangePasswordControl {
     this.$submittableInputs.removeAttr('required');
     this.$inputsBlock.find('input').val('');
     this.$inputsBlock.find('.form-text').text('');
+    $('#employee_change_password_new_password_first').click();
```

Comment:
I'm not sure this is part of your PR initially Maybe this as added after your rebase? But it's a global component that doesn't seem related to the Country form Besides it doesn't use a selector from a selector map, that's suspicious 🤔

Reply:
This should be:
```suggestion
    this.$newPasswordInputs.click();
```

But again I don't understand what it's doing in this PR?